sdks/java/core/src/main/java/org/apache/beam/sdk/testing/WindowFnTestUtils.java

```diff
@@ -67,14 +68,30 @@
   public static <T, W extends BoundedWindow> Map<W, Set<String>> runWindowFn(
       WindowFn<T, W> windowFn,
       List<Long> timestamps) throws Exception {
```

Comment:
Yes I totally agree, I did not change it to keep the same test API. But I can change it to `Instant` and update all the tests that use WindowFnTestUtils